pkg/validators/validators.go

```diff
@@ -49,6 +51,22 @@ func handleClientError(e error) error {
 	return e
 }
 
+// TestDeploymentVariablesNotUsed errors if there are any unused deployment
+// variables and prints any to the output for the user
+func TestDeploymentVariablesNotUsed(unusedVariables []string) error {
+	var foundUnused bool
+	for _, v := range unusedVariables {
+		foundUnused = true
+		log.Printf(unusedDeploymentVariableMsg, v)
+	}
+
+	if foundUnused {
```

Comment:
No need for `foundUnused`, just use `len(unusedVariables) > 0`

Reply:
Addressed.